processes/security_team_members.md

```diff
@@ -0,0 +1,63 @@
+# Node.js Security Team
+
+Node.js security teams are expected to keep all information that they have
+privileged access to by being on the team completely private to the team. This
+includes notifying anyone outside the team of issues that have not yet been
+disclosed publically, including the existence of issues, expectations of
+upcoming releases, and patching of any issues other than in the process of
+their work as a member of the security team.
+
+Membership on the security teams can be requested via an issue in the TSC repo,
+and must be approved by current team members.
+
+Members of the security teams should indicate that they accept the privacy
+policies by PRing their acceptance to this file.
+
+## Team that triages security reports
```

Comment:
Should we explicitly define the scope of the security team to nodejs core only?

Reply:
I think so, we probably need 2 of these.  One for Node core and then a different one for modules.

Reply:
My thinking was we would document here the _current_ team and practice. Once we have a thirdparty triage team and process, we can add that. But, I can add more future looking wording if that works.

Reply:
So then does this doc explicitly cover only security reports and issues in node-core? If so, +1 to adding wording in the doc to making it explicit- and agree with @mhdawson that in the future there are probably separate teams for core vs modules

Reply:
@sam-github I agreed, probably just good to make it clear that in this one its focused on core

Reply:
+1 to explicitly defining core vs. module vulnerabilities.